WordPress/src/test/java/org/wordpress/android/ui/accounts/login/jetpack/LoginNoSitesViewModelTest.kt

```diff
@@ -152,11 +153,11 @@ class LoginNoSitesViewModelTest : BaseUnitTest() {
     }
 
     private fun setupInstanceStateForNoUser() {
-        whenever(savedInstanceState.getSerializable(KEY_STATE)).thenReturn(NoUser)
+        whenever(savedInstanceState.getSerializableCompat<LoginNoSitesViewModel.State?>(KEY_STATE)).thenReturn(NoUser)
```

Comment:
Minor (🔍): Consider removing the nullable `?` from the `LoginNoSitesViewModel.State` compat extension type.

PS: Same applies to the below private test function.

Reply:
Without `?`, tests were failing with the following message. 
```
null cannot be cast to non-null type org.wordpress.android.ui.accounts.login.jetpack.LoginNoSitesViewModel.State
```

I also added `?` to Serializable in `getSerializableCompat()` function to fix it.
`inline fun <reified T : Serializable?> Bundle.getSerializableCompat(key: String): T?`

I couldn't find the actual reason why it casts null to non-null. I suspect it could be Mock library. Adding `?` was the best solution I could find.

Reply:
Oh great, thanks for the explanation on that @irfano , this makes sense and I actually didn't run the tests before leaving that comment here, thank you for being thorough! 🙇 ❤️